bob/src/server.rs

```diff
@@ -217,8 +217,13 @@ where
         }
     }
 
-    async fn ping(&self, _: Request<Null>) -> ApiResult<Null> {
+    async fn ping(&self, r: Request<Null>) -> ApiResult<Null> {
         debug!("PING");
+        if let Some(node_name) = r.metadata().get("node_name") {
```

Comment:
`node_name` was used for authentication and the behavior has changed. You should add `set_node_name` function to [bob_client.rs](https://github.com/qoollo/bob/blob/master/bob-common/src/bob_client.rs), that will set the name to metadata